cmake/modules/SwiftComponents.cmake

```diff
@@ -122,3 +122,29 @@ function(swift_install_in_component component)
     install(${ARGN})
   endif()
 endfunction()
+
+function(swift_install_symlink_component component)
+  cmake_parse_arguments(
+      ARG # prefix
+      "" "LINK_NAME;TARGET;DESTINATION" "" ${ARGN})
```

Comment:
Chris, one CMake utility that would be useful is a wrapper around cmake_parse_arguments that is self documenting. I.e.:

swift_parse_arguments(
   PREFIX Arg
   FLAGS ""
   OPTIONS "LINK_NAME;TARGET;DESTINATION"
   MULTIVALUE_OPTIONS ""
   ARGS ${ARGN}).

Thoughts?